[2023-06-21T20:25:58.910+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: showpulse_dbt_dag.dbt_run scheduled__2023-06-21T20:24:54.561634+00:00 [queued]>
[2023-06-21T20:25:58.916+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: showpulse_dbt_dag.dbt_run scheduled__2023-06-21T20:24:54.561634+00:00 [queued]>
[2023-06-21T20:25:58.916+0000] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2023-06-21T20:25:58.917+0000] {taskinstance.py:1289} INFO - Starting attempt 1 of 1
[2023-06-21T20:25:58.918+0000] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2023-06-21T20:25:58.925+0000] {taskinstance.py:1309} INFO - Executing <Task(BashOperator): dbt_run> on 2023-06-21 20:24:54.561634+00:00
[2023-06-21T20:25:58.929+0000] {standard_task_runner.py:55} INFO - Started process 6055 to run task
[2023-06-21T20:25:58.932+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'showpulse_dbt_dag', 'dbt_run', 'scheduled__2023-06-21T20:24:54.561634+00:00', '--job-id', '986', '--raw', '--subdir', 'DAGS_FOLDER/dbt_run.py', '--cfg-path', '/tmp/tmp5zdqf3t9']
[2023-06-21T20:25:58.933+0000] {standard_task_runner.py:83} INFO - Job 986: Subtask dbt_run
[2023-06-21T20:25:58.987+0000] {task_command.py:389} INFO - Running <TaskInstance: showpulse_dbt_dag.dbt_run scheduled__2023-06-21T20:24:54.561634+00:00 [running]> on host 62cfed75eae1
[2023-06-21T20:25:59.035+0000] {taskinstance.py:1518} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=nburkett
AIRFLOW_CTX_DAG_ID=showpulse_dbt_dag
AIRFLOW_CTX_TASK_ID=dbt_run
AIRFLOW_CTX_EXECUTION_DATE=2023-06-21T20:24:54.561634+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-06-21T20:24:54.561634+00:00
[2023-06-21T20:25:59.037+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-06-21T20:25:59.037+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'cd /dbt/ticketmaster_transformation && dbt run']
[2023-06-21T20:25:59.045+0000] {subprocess.py:86} INFO - Output:
[2023-06-21T20:26:00.271+0000] {subprocess.py:93} INFO - [0m20:26:00  Running with dbt=1.5.1
[2023-06-21T20:26:00.765+0000] {subprocess.py:93} INFO - [0m20:26:00  Found 4 models, 0 tests, 0 snapshots, 0 analyses, 356 macros, 0 operations, 0 seed files, 3 sources, 0 exposures, 0 metrics, 0 groups
[2023-06-21T20:26:00.768+0000] {subprocess.py:93} INFO - [0m20:26:00
[2023-06-21T20:26:01.418+0000] {subprocess.py:93} INFO - [0m20:26:01  Concurrency: 4 threads (target='dev')
[2023-06-21T20:26:01.421+0000] {subprocess.py:93} INFO - [0m20:26:01
[2023-06-21T20:26:01.435+0000] {subprocess.py:93} INFO - [0m20:26:01  1 of 4 START sql table model twitter_kafka_pyspark_test.distinct_data .......... [RUN]
[2023-06-21T20:26:01.440+0000] {subprocess.py:93} INFO - [0m20:26:01  3 of 4 START sql table model twitter_kafka_pyspark_test.my_first_dbt_model ..... [RUN]
[2023-06-21T20:26:01.445+0000] {subprocess.py:93} INFO - [0m20:26:01  2 of 4 START sql table model twitter_kafka_pyspark_test.dm_events .............. [RUN]
[2023-06-21T20:43:30.017+0000] {local_task_job.py:277} WARNING - State of this instance has been externally set to failed. Terminating instance.
[2023-06-21T20:43:30.021+0000] {process_utils.py:133} INFO - Sending Signals.SIGTERM to group 6055. PIDs of all processes in the group: [6056, 6055]
[2023-06-21T20:43:30.022+0000] {process_utils.py:84} INFO - Sending the signal Signals.SIGTERM to group 6055
[2023-06-21T20:43:30.026+0000] {taskinstance.py:1488} ERROR - Received SIGTERM. Terminating subprocesses.
[2023-06-21T20:43:30.028+0000] {subprocess.py:104} INFO - Sending SIGTERM signal to process group
[2023-06-21T20:43:30.162+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=6056, status='terminated', started='20:25:58') (6056) terminated with exit code None
[2023-06-21T20:43:30.163+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=6055, status='terminated', exitcode=0, started='20:25:57') (6055) terminated with exit code 0
